src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_telemetry.py

```diff
@@ -0,0 +1,61 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import time
+import yaml
+from azure.cli.command_modules.containerapp._utils import format_location
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only, StorageAccountPreparer)
+
+from .common import TEST_LOCATION, STAGE_LOCATION
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class ContainerappEnvTelemetryScenarioTest(ScenarioTest):
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northeurope")
+    def test_containerapp_env_telemetry_e2e(self, resource_group):
+        self.cmd('configure --defaults location={}'.format('eastus'))
+
+        env_name = self.create_random_name(prefix='containerapp-env', length=24)
+        ai_conn_str = f'InstrumentationKey={self.create_random_name(prefix="ik", length=8)};IngestionEndpoint={self.create_random_name(prefix="ie", length=8)};LiveEndpoint={self.create_random_name(prefix="le", length=8)}'
+        data_dog_site = self.create_random_name(prefix='dataDog', length=16)
+        data_dog_key = self.create_random_name(prefix='dataDog', length=16)
+        traces_destinations = "appInsights"
+        logs_destinations = "appInsights"
+        metrics_destinations = "dataDog"
+
+        logs_workspace_name = self.create_random_name(prefix='containerapp-env', length=24)
+
+        logs_workspace_id = self.cmd('monitor log-analytics workspace create -g {} -n {} -l eastus'.format(resource_group, logs_workspace_name)).get_output_in_json()["customerId"]
```

Comment:
Do the case really need logs workspace? If not, you can use `--logs-destination none` .